frontend/packages/kubevirt-plugin/src/components/vm-snapshots/vm-snapshots.tsx

```diff
@@ -81,17 +89,33 @@ export const VMSnapshotsPage: React.FC<VMLikeEntityTabProps> = ({ obj: vmLikeEnt
   const vmName = getName(vmLikeEntity);
   const namespace = getNamespace(vmLikeEntity);
 
-  const resource: WatchK8sResource = React.useMemo(
-    () => ({
-      isList: true,
-      kind: VirtualMachineSnapshotModel.kind,
-      namespaced: true,
-      namespace,
-    }),
-    [namespace],
-  );
+  const snapshotResource: WatchK8sResource = {
+    isList: true,
+    kind: VirtualMachineSnapshotModel.kind,
+    namespaced: true,
+    namespace,
+  };
+
+  const restoreResource: WatchK8sResource = {
+    isList: true,
+    kind: VirtualMachineRestoreModel.kind,
+    namespaced: true,
+    namespace,
+  };
 
-  const [snapshots, snapshotsLoaded, snapshotsError] = useK8sWatchResource<VMSnapshot[]>(resource);
+  const [snapshots, snapshotsLoaded, snapshotsError] = useK8sWatchResource<VMSnapshot[]>(
+    snapshotResource,
+  );
+  const [restores, restoresLoaded, restoresError] = useK8sWatchResource<VMRestore[]>(
+    restoreResource,
+  );
+  const sortedRestores = React.useMemo(
```

Comment:
it might be easier to create a map with `getVmRestoreSnapshotName` key here for better consumption. 
Then it wouldn't need to be even sorted as you could just save the relevant restore.

Reply:
which snapshotName? one vm can have multiple snapshots with different names? a row for each.

Reply:
yes, but relevant restore shoul be only one , right?